pkg/kubernetes/client.js

```diff
@@ -415,7 +415,7 @@ define([
 
                     update_error_message(ex, response);
                     var msg = "watching kubernetes " + type + " failed: " + ex.message;
-                    if (ex.problem !== "disconnected")
+                    if (ex.problem !== "disconnected" && ex.problem !== "not-found")
```

Comment:
This doesn't stop the message for me, ex.problem can be null for 404s. Below we are checking ex.status to determine if it's a 404 that might work here too.